dts/bindings/clock/st,stm32wb-rcc.yaml

```diff
@@ -9,8 +9,6 @@ compatible: "st,stm32wb-rcc"
 
 include:
   - name: st,stm32-rcc.yaml
-    property-blocklist:
-      - ahb-prescaler
```

Comment:
Why removing this ?
I think, this is still  valid, ahb-prescaler does not exist on wb series

Though, the modification to be done is to extend enum on cpu1-prescaler

Reply:
Ok, I see what you mean